pkg/clusterapi/extraargs.go

```diff
@@ -102,6 +102,13 @@ func ControlPlaneNodeLabelsExtraArgs(cpc v1alpha1.ControlPlaneConfiguration) Ext
 	return nodeLabelsExtraArgs(cpc.Labels)
 }
 
+// CgroupDriverExtraArgs args added for kube versions below 1.24.
+func CgroupDriverExtraArgs() ExtraArgs {
+	args := ExtraArgs{}
+	args.AddIfNotEmpty("cgroup-driver", "cgroupfs")
```

Comment:
Did we check if the default is systemd for all kube versions we support? Wondering if we want to choose between cgroupfs and systemd instead of omitting

Reply:
yes from 1.24 onwards it defaults to systemd, confirmed that with testing.